.gitignore

```diff
@@ -0,0 +1,27 @@
+<<<<<<< HEAD
```

Comment:
This line is part of a conflict and should be removed if the conflict was resolved

Reply:
^^^ @gtl90